modules/core/src/utils/math-utils.ts

```diff
@@ -132,3 +132,11 @@ export function toDoublePrecisionArray(
 
   return scratchArray.subarray(0, count * size * 2);
 }
+
+type LayerBounds = [number[], number[]];
+export function mergeBounds(bounds1: LayerBounds, bounds2: LayerBounds): LayerBounds {
```

Comment:
I have an idea. What if we change the base layer `getBounds` signature to

```js
getBounds(attributes?: string[]): [number[], number[]] | null
```

Then from the ArcLayer you can just use

`super.getBounds(['instanceSourcePositions', 'instanceTargetPositions'])`

Without having to export a new util.